test/erc721.js

```diff
@@ -137,8 +119,7 @@ describe('ERC721', () => {
 				const deployContractPromise = deployedContract.call('mint', {
 					args: `(${firstTokenId}, ${config.pubKeyHex})`,
 					options: {
-						ttl: config.ttl,
-						gas: config.gas
+						ttl: config.ttl
 					},
 					abi: "sophia"
```

Comment:
'options' repeats to many times .... can you extract it ..you can pass your params to extracted function ....if you want to change something you should do more work ... just like now .